View/PassengerView.cs

```diff
@@ -0,0 +1,272 @@
+            { 
+                sb.AppendLine(ticket.ToString());
+                sb.AppendLine("****************");
+            }
+            sb.AppendLine();
+            Console.WriteLine(sb.ToString());
+        }
+    }
+    public void DisplayCancelBooking() 
+    {
+        if(passengerviewModel.ViewBookings() is null)
+        Console.WriteLine("Enter the id of the tecket you need to cancle:");
+        var ticketID = Console.ReadLine();
+        var boockings = passengerviewModel.ViewBookings();
+        if(!String.IsNullOrEmpty(ticketID) && long.TryParse(ticketID, out var ID))
+        {
+            var deletedBooking = passengerviewModel.CancelTicket(null,ID);
+            if(deletedBooking != null)
+            {
+                Console.WriteLine(deletedBooking.ToString());
+                Console.WriteLine($"The Booking with the ID: {ID} has been deleted");
+                Console.WriteLine();
+            }
+            else
+            {
+                Console.WriteLine($"There's no Booking with the ID: {ID}");
+                Console.WriteLine();
+            }
+                
+        }
+        //Todo: cancel be creating new ticket
+     
+    }
+    public void DisplayUpdateBooking() 
+    {
+        Console.WriteLine("Enter the id of the ticket to update");
+        var ticketID = Console.ReadLine();
+        if(String.IsNullOrEmpty(ticketID) || !long.TryParse(ticketID, out var ID))
+        {
+            Console.WriteLine($"Invalid input: {ticketID}");
+            return;
+        }
+        Console.WriteLine("Here are the abalibale flights:");
+        var flights = passengerviewModel.FillterFlights(null, null, null, null, null, null, null);
+        if(flights.Any() )
+        {
+            StringBuilder sb = new();
+            foreach( var _flight in flights )
+            {
+                sb.AppendLine(_flight.ToString());
+            }
+            Console.WriteLine(sb.ToString());
+            Console.WriteLine();
+        }
+        else
+        {
+            Console.WriteLine("There are no abalibale flights.");
+            return;
+        }
+        Console.WriteLine("Enter the Flight id:");
+        var validFlightID = long.TryParse(Console.ReadLine(), out var flightID);
+        if(!validFlightID)
+        {
+            Console.WriteLine("invalid id format.");
+            return;
+        }
+        var flight = passengerviewModel.FindFlightByID(flightID);
+        if(flight == null)
+        {
+            Console.WriteLine($"There are no flights with ID: {flightID}");
+            return;
+        }
+        Ticket ticket = new() { Flight = flight.Id,
+            Passenger = _user.Id,
+            DepartureAirport = flight.DepartureAirport,
+            DestinationAirport = flight.DestinationAirport,
+            Time = flight.Time
+        };
+        passengerviewModel.UpdateBooking(ID, ticket);
+
+    }
+    public void DisplayFilterFlights()
```

Comment:
Can we rewrite this Like maybe use private methods to make it easier to read, also conditions are not straightforward can we name them I mean use variable